scss/contact/_contact-list.scss

```diff
@@ -111,14 +111,13 @@ div.create-chat-contact-list-wrapper {
 
       .DisplayName {
         line-height: 68px;
-        margin-left: -24px;
-        padding-left: 30px;
-        padding-right: 22px;
+        margin-left: -33px;
+        padding-left: 39px;
+        padding-right: 12px;
         height: var(--local-height);
-        background-color: #e6e6e6;
+        background-color: #8080806b;
```

Comment:
I dislike the color change on light theme, can you use that color specifically on the dark theme?